.github/workflows/check-prdoc.yml

```diff
@@ -48,8 +48,8 @@ jobs:
         run: |
           hits=$(find prdoc -name "pr_$GITHUB_PR*.prdoc" | wc -l)
           if (( hits > 0 )); then
-            echo "PR detected as silent, but a PRDoc was found, checking it as information"
-            $ENGINE run --rm -v $PWD:/repo $IMAGE check -n ${GITHUB_PR} || echo "Ignoring failure"
+            echo "PR detected as silent, but a PRDoc was found, checking it"
+            $ENGINE run --rm -v $PWD:/repo $IMAGE check -n ${GITHUB_PR}
```

Comment:
This step checks only if there is a PRdoc for this PR and validates it. If the idea was to check all existing prdocs I'd suggest adding an additional job `check-prdoc-all` that will check all prdocs.

Reply:
yes, in this commit I refactor to just check for all prdoc:
https://github.com/paritytech/polkadot-sdk/pull/7543/commits/6930f833f5d4aaa371b9de2f052b96e27e27e61a